modules/game/server/sockets/game.server.socket.config.js

```diff
@@ -267,8 +268,7 @@ module.exports = function (io, socket) {
         numDrawers: 1,
         roundTime: GameSettings.roundTime.default,
         timeAfterGuess: GameSettings.timeAfterGuess.default,
-        topicListName: TopicSettings.topicListName.default,
-        topicListDifficulty: TopicSettings.topicListDifficulty.default
+        topicListName: 'English (easy)',
```

Comment:
This is kind of against the comment I left earlier...here I *am* setting an arbitrary default which should exist but may not.

Thoughts?